docker/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN mkdir -p /scilla/0/bin2/ && cp -L /scilla/0/bin/* /scilla/0/bin2/ && strip /
 FROM zilliqa/zilliqa-ccache:${ZILLIQA_CCACHE_VERSION} AS builder
 
 # Manually input tag or commit, can be overwritten by docker build-args
-ARG COMMIT_OR_TAG=master
```

Comment:
should be `v8.4.0` , right?

Reply:
Yes.. it's usually passed in as a --build-arg though in the command line though.

Reply:
I don't know if this is correct - `8.4.0` isn't a tag in GitHub. Really I think we should just leave this blank if we're expecting it to be passed as a build-arg?